packages/components/app/styles/components/side-nav/main.scss

```diff
@@ -66,56 +72,100 @@
 
 .hds-side-nav__menu-toggle-button {
   position: absolute;
+  top: -2px;
+  left: calc(var(--hds-app-sidenav-border-width) * -1);
   z-index: 1;
+  display: flex;
+  box-sizing: content-box;
+  padding: calc(var(--token-side-nav-toggle-button-padding-vertical) - var(--hds-app-sidenav-border-width)) var(--token-side-nav-toggle-button-padding-horizontal);
   color: var(--token-color-foreground-high-contrast);
+  text-align: right;
   background: none;
-  border: 1px solid transparent;
+  background-color: var(--token-side-nav-color-surface-primary);
+  border: var(--hds-app-sidenav-border-width) solid var(--hds-app-sidenav-border-color);
+  border-left-color: transparent;
+  border-top-right-radius: var(--hds-app-sidenav-border-radius);
```

Comment:
this variable is used only in this context, not across the entire sidenav. maybve better to rename it to `--hds-app-sidenav-toggle-button-border-radius`?

Reply:
Renamed in 862e086318e37a9812cae807303e57546b928e0c